#### iGibson Simulator Parameters

robot_ns: "jackalJaco"

### NUA TODO: DEPRECATE THIS OR CURRENT YAML!
mobiman_config_path: "config/drl/config_mobiman_drl.yaml"

# headless
# gui_interactive
mode: "headless"
use_pb_gui: true

action_timestep: 0.01
physics_timestep: 0.01

# Scene
scene: "empty"
floor_plane_rgba: [0.6, 0.6, 0.6, 1]

# Robot
robot_name: "JackalJaco"

# Task
task: mobiman_pick

### NUA TODO: DEFINE THESE IN A SCRIPT THAT DEFINE IGIBSON TASK!
## Objects
objects: {conveyor_belt: 101_conveyor, 
          red_cube: 102_cube}

conveyor_pos: [0.0, 2.5, 0.1]